pkg/plugins/plugin.go

```diff
@@ -24,7 +24,7 @@ var (
 	// NamePattern is the plugin name regex pattern
 	NamePattern = regexp.MustCompile(`^[0-9a-zA-Z-_]+$`)
 	// VersionPattern is the plugin version regex pattern
-	VersionPattern = regexp.MustCompile(`^[0-9a-zA-Z+\\.-]+$`)
+	VersionPattern = regexp.MustCompile(`^[0-9a-zA-Z+\\.-_]+$`)
```

Comment:
```suggestion
	VersionPattern = regexp.MustCompile(`^[0-9a-zA-Z+\\.\-_]+$`)
```

I believe you need to escape the dash since otherwise it's treated as a range between . and _